pkg/hintrunner/hint.go

```diff
@@ -215,3 +215,40 @@ func (hint WideMul128) Execute(vm *VM.VirtualMachine) error {
 
 	return nil
 }
+
+type SquareRoot struct {
+	value ResOperander
+	dst   CellRefer
+}
+
+func (hint SquareRoot) String() string {
+	return "SquareRoot"
+}
+
+func (hint SquareRoot) Execute(vm *VM.VirtualMachine) error {
+	value, err := hint.value.Resolve(vm)
+	if err != nil {
+		return fmt.Errorf("resolve value operand %s: %v", hint.value, err)
+	}
+
+	valueFelt, err := value.FieldElement()
+	if err != nil {
+		return err
+	}
+
+	sqrt := valueFelt.Sqrt(valueFelt)
+
+	dst := f.Element{}
+	dst.Set(sqrt)
+
+	dstAddr, err := hint.dst.Get(vm)
+	if err != nil {
+		return fmt.Errorf("get destination cell: %v", err)
+	}
+	dstVal := memory.MemoryValueFromFieldElement(&dst)
```

Comment:
Why do you create a `dst` which is equal to `sqrt` felt and then store `dst` without any further changes. Why not storing `sqrt` directly?